modules/sno-du-managing-network-secrets.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-managing-network-secrets_{context}"]
+= Managing network secrets
```

Comment:
The ClusterDeployment CR

Reply:
@imiller0 Discuss

Reply:
Stephen to Change title to Configuring cluster installation using the ClusterDeployment CR

Reply:
changed.